.gitignore

```diff
@@ -9,7 +9,6 @@ models/*
 dist/*
 *.tar.gz
 config.yaml
-setup.py
```

Comment:
can you keep this and just allow the setup template to be changed?

`! ml_pipeline_gen/templates/setup.py`

Reply:
We do want to ignore the setup.py that is generated by the template, which is why this line is needed.

Reply:
ack, updated in new commit